index.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Alick's Portfolio</title>
+  <meta name="Alick Nyirenda" content="Alick Portfolio">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <p class="header-text">Alick Nyirenda </p>
+    <a href="#"><i class="header-icon fa fa-bars fa-lg"></i></a>
+  </header>
+  <main>
+  <section class="headline">
+    <h1 class="headline-text-1">Alick Nyirenda</h1>
+    <p class="headline-text-2">Software Developer</p>
+  </section>
+  <section class="headline-2">
+    <h3 class="headline-text-3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</h3>
```

Comment:
- [ ] This is not a heading or title, as it is a paragraph it should be a `p` tag. 
_See my comments above about semantic HTML._